.github/workflows/maven.yml

```diff
@@ -35,13 +35,14 @@ jobs:
         if: contains(matrix.os, 'win') == false
         run: chmod +x ./mvnw
       - name: Build with Maven
-        run: ./mvnw install --file pom.xml --batch-mode
+        # `-DskipTests=true` since they run on `coverage` job.
+        run: ./mvnw install --file pom.xml --batch-mode -DskipTests=true
```

Comment:
Tests already run in the `coverage` job, so they were duplicated here.

Reply:
Coverage only runs the tests against one operating system and with the latest JDK.

These matrix tests ensure that everything works okay on all supported platforms. Please revert.

Reply:
Done in e0bb83e.